docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1089,6 +1107,18 @@ Default stratifications to all observers should include scenario and input draw.
       * Check that coverage at each facility type is as expected
     - 
     - 
+  * - 7.1
+    - * Neonatal mortality (all cause and cause-specific) is expected to remain slightly underestimated in the baseline scenario (by the same magnitude of model run 6.1). This is expected as we have not yet implemented a strategy to account for known probabilities greater than 1.
+      * Recheck LBWSG Effects
+      * Check that intervention effect sizes are maintained
+    - 
+    - 
+  * - 7.2
+    - * Neonatal mortality is expected to validate to GBD targets (no longer be underestimated!)
+      * Check that LBWSG effects are updated and functioning as expected
+      * Check that intervention effect sizes are maintained too
+    - 
+    - 
```

Comment:
@aflaxman here is the V&V criteria for reference.

Reply:
I think we should update [V&V criteria on the neonatal mortality page](https://github.com/ihmeuw/vivarium_research/pull/1654/files#diff-b271710fb04839c6836406d54b945d2c223aaf854a55bf6d330eccbfccdc6ed2R361) too, since it says we're using a mortality hazard, which we no longer are.